docs/index.md

```diff
@@ -6,107 +6,53 @@ permalink: /docs/index.html
+| [encoding](encoding.html) | Object        | key-value mapping between encoding channels and encoding object |
 | [config](config.html)   | Object        | Configuration object. |
 
-## Using Vega-Lite
-
-### Inline Data
+## Basic usage
 
-Here is the bare minimum html file to get Vega-Lite with inline values working in a webpage.
-Basically, Vega-Lite compiles a Vega-Lite specification into a Vega
-specification and use Vega Runtime to display visualizations.
+To use Vega-Lite, load the required libraries (D3, Vega, and Vega-Lite).
 
 ```html
-<!DOCTYPE html>
-<title>Vega Lite Bar Chart</title>
-<meta charset="utf-8">
-
-<script src="../lib/d3.min.js"></script>
-<script src="../lib/vega.js"></script>
-<script src="../lib/vega-lite.js"></script>
-
-<div id="vis"></div>
-
-<script>
-
-var vlSpec = {
-      "data": {
-        "values": [
-          {"a":"A", "b":28}, {"a":"B", "b":55}, {"a":"C", "b":43},
-          {"a":"D", "b":91}, {"a":"E", "b":81}, {"a":"F", "b":53},
-          {"a":"G", "b":19}, {"a":"H", "b":87}, {"a":"I", "b":52}
-        ]
-      },
-      "mark": "bar",
-      "encoding": {
-        "x": {"type": "ordinal","field": "a"},
-        "y": {"type": "quantitative","field": "b"}
-      }
-    };
-
-var vgSpec = vl.compile(vlSpec).spec;
-vg.parse.spec(vgSpec, function(chart) {
-  chart({el:"#vis"}).update(); });
-
-</script>
+<script src="//d3js.org/d3.v3.min.js"></script>
+<script src="//vega.github.io/vega/vega.js"></script>
+<script src="//vega.github.io/vega-lite/vega-lite.js"></script>
 ```
 
+To compile a Vega-Lite specification to Vega, call `vl.compile`.
 
-### Data from URL
-
-Here is the bare minimum html file to get Vega-Lite with data from url working in a webpage.
-
-```html
-<!DOCTYPE html>
-<title>Vega Lite Bar Chart</title>
-<meta charset="utf-8">
-
-<script src="../lib/d3.min.js"></script>
-<script src="../lib/vega.js"></script>
-<script src="../lib/vega-lite.js"></script>
-
-<div id="vis"></div>
-
-<script>
-
-var vlSpec = {
-      "data": {"url": "data/cars.json"},
-      "mark": "point",
-      "encoding": {
-        "x": {"type": "ordinal","field": "Origin"},
-        "y": {"type": "quantitative","field": "Acceleration"}
-      }
-    };
+```js
 var vgSpec = vl.compile(vlSpec).spec;
+```
+
+Then render the Vega specification with the [Vega runtime](https://github.com/vega/vega/wiki/Runtime).
 
+```js
 vg.parse.spec(vgSpec, function(chart) {
-  var view = chart({el: '#vis', renderer: 'svg'});
-  view.update();
+  chart({el:"#vis"}).update();
 });
-</script>
 ```
 
+## Vega-Lite, Vega, and D3
 
-__Pending Revision__:
-Vega-Lite Compile API is under revision.  A better tutorial is coming soon.
+Vega-Lite is a higher-level grammar for visual analysis. Common charts (bar chart, line chart, area chart, scatter plot, heatmap, trellis plots, ...) can be easily created with Vega-Lite, often in a few lines of JSON. Vega is much more expressive and also supports interactions.
```

Comment:
I feel like you should hide this paragraph for now. 
We should talk about hierarchy of defaults - Vega-LIte constructs scales, legends and axes by default and also have a slightly higher-level of mark type (bar rather than rect) -- but we better polish other parts first.